crates/db_schema/src/schema.rs

```diff
@@ -162,6 +162,7 @@ table! {
         show_new_post_notifs -> Bool,
         email_verified -> Bool,
         accepted_application -> Bool,
+        discussion_languages -> Array<Text>,
     }
```

Comment:
SQL rule #1: never use arrays in a column.

Reply:
I dont think we should blindly follow such a rule. In this case an array makes a lot of sense, because the items are tightly coupled with the local_user table, and wont have relations to any other tables. Also there is no reason to access the elements one by one, but always together.

https://stackoverflow.com/q/20219503

https://stackoverflow.com/q/6843912

Reply:
See comment above. You have to use an intermediary table to avoid the eq_in query, which it isn't possible to index, and are always going to be slow.

In general you should never use arrays in columns in SQL (goes against all the original intentions of normalized data vectors). The one case I could see using them, is if you aren't using them to join anything, but that isn't the case here.